src/components/AuthModal/AuthModal.scss

```diff
@@ -0,0 +1,79 @@
+.backdrop-div {
```

Comment:
Wrap you up a parent class

Reply:
The `backdrop-div` class is on the parent `div` of the `AuthModal` component. Perhaps I should rename the class to `auth-modal-div`?